plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -326,6 +343,39 @@ SecRule REQUEST_FILENAME "@contains /remote.php/dav/files/" \
         "t:none,\
         ctl:ruleRemoveById=950100"
 
+# Uploading files to Nextcloud
+# This FP is usually triggered when uploading via the mobile app
+SecRule REQUEST_FILENAME "@rx /remote\.php/dav/uploads/[^/]+/[0-9]+/[0-9]+$" \
+    "id:9508174,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
+    ctl:ruleRemoveById=920341,\
+    ctl:ruleRemoveTargetById=944200;REQUEST_BODY,\
```

Comment:
Please order by ctl action, then ID.

Reply:
I'm not sure what you mean, do you mean doing something like this?
```
    ctl:ruleRemoveTargetById=944200;REQUEST_BODY,\
    ctl:ruleRemoveById=920341"
```

Reply:
No, sorry. I just wanted you to sort by ID, but by ctl action first, so that all rules with the same ctl action are grouped together, not the other way around (in this case that doesn't matter though, I had just copied my comment :) ).

Reply:
gotcha, lots of rules in the plugin aren't ordered in that manner so i'll change those as well